sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -284,6 +284,21 @@
   /** Controls if the `tracestate` header is attached to envelopes and HTTP client integrations. */
   private boolean traceSampling;
 
+  /** The cache dir. path for caching profiling traces */
+  private @Nullable String profilingTracesDirPath;
```

Comment:
Should some of these properties be on `SentryAndroidOptions` or do we intend to expand to Java backend/desktop too?

Reply:
Ok, will move the profiling options there

Reply:
I moved `profilingTracesDirPath` and `profilingTracesIntervalMillis` to `SentryAndroidOptions`
I kept `profilingEnabled`, `maxTraceFileSize` and `transactionListener` in `SentryOptions`